app/javascript/application.js

```diff
@@ -1,15 +1,16 @@
 // Entry point for the build script in your package.json
 import "@hotwired/turbo-rails"
 import "./controllers"
-
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 
-function App() {
-  return (<h1>Hello World!</h1>);
-}
+import App from "./App";
+import store from "./Redux/store";
 
 ReactDOM.render(
-  <App/>,
+  <Provider store={store}>
+    <App/>
+  </Provider>,
```

Comment:
- [ ] Kindly make sure your two apps correctly work together after creating `greetings` in your database. Right now, user can create greetings but the nothing can't be displayed to the to the **UI**. I suggest you check your react-route configuration

Reply:
Has been resolved by adding a new controller